ArduPlane/RC_Channel.cpp

```diff
@@ -443,6 +444,21 @@ bool RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const AuxSwit
         }
         break;
 
+    case AUX_FUNC::TECS_PROP_FAILED:
```

Comment:
gate behind a new `AP_RC_CHANNEL_PROPULSION_FAILED_ENABLED` as we do for killing IMUS.  Disable on 1MB boards, and consider just outright disabling it.

Reply:
Yeah, working on the define now

Reply:
Added. Had to rework this a bit so that I would generate something that could be found by extract_features.py

Weirdly, despite defaulting to enabled, this PR is somehow _saving_ flash compared to master. It doesn't make sense.

Disabling saves another ~130 bytes still.